ONSdigital/response-operations-ui
# RAS-1088 Implement 48 hour grace period for survey change of status

# What and why?
Adds a 48 hour grace period to the survey status change (from complete to not started) 
# How to test?
- deploy this pr into your environment
- run the acceptance tests
- complete the survey in frontstage
- go to `/case/49900000001/response-status?survey=QBS&period=1912`
- make sure the radio to reset to Not Started is disabled
- go to the caseevent table in the case db and edit the date of the response (`OFFLINE_RESPONSE_PROCESSED`) to have happened at lease 48 hours ago
- refresh `/case/49900000001/response-status?survey=QBS&period=1912`
- make sure the radio is now enabled and you can reset the status
# Jira
[RAS-1086](https://jira.ons.gov.uk/browse/RAS-1086)
